static/components/PoiCard.js

```diff
@@ -9,8 +9,10 @@ class PoiCard extends React.Component {
     super(props)
     this.state = {
       from: {},
-      to: {}
+      to: {},
     }
+    this.hoverCard = this.hoverCard.bind(this)
```

Comment:
Would you just mind adding a comment like:
```
// cf: https://facebook.github.io/react/docs/handling-events.html
```

It might be obvious for React dev maybe, but we're not there yet ;)